cumulusci/tasks/salesforce/PublishCommunity.py

```diff
@@ -0,0 +1,60 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import SalesforceException
+
+
+class PublishCommunity(BaseSalesforceApiTask):
+    api_version = "46.0"
+    task_docs = """
+    Publishes a Salesforce Community via the Connect API.
+    """
+    task_options = {
+        "name": {
+            "description": "The name of the community to publish.",
+            "required": True,
+        },
+        "communityid": {
```

Comment:
Can we do `snake_case` for this parameter?

I'd suggest we make either Id or Name required - i.e., throw a `TaskOptionsError` if neither is provided, but allow the user to pick which one.

Reply:
@davidmreed Incorporated both suggestions.